projects/packages/videopress/src/client/lib/url/index.ts

```diff
@@ -129,3 +129,44 @@ export function buildVideoPressURL(
 export const removeFileNameExtension = ( name: string ) => {
 	return name.replace( /\.[^/.]+$/, '' );
 };
+
```

Comment:
VideoPress package would be excellent and something to consider in the near future. We already have other packages in the Jetpack ecosystem, so I don't see why not. It will require more time, of course. :-)